packages/vscode-extension/tests/webviewUtils.ts

```diff
@@ -212,3 +216,18 @@ export async function executeFile(
   await openFixtureFile(browser, fileName, opts);
   await workbench.executeCommand('neo4j.runCypher');
 }
+
+export async function ensureNotificationsAreDismissed(
+  browser: WebdriverIO.Browser,
+): Promise<void> {
+  const wb = await browser.getWorkbench();
+  const notifications = await wb.getNotifications();
+  for (const notification of notifications) {
+    await notification.dismiss();
+  }
+
+  const remainingNotifications = await wb.getNotifications();
```

Comment:
Interesting. Why do we need to get notifications twice?

Reply:
In my testing overflowing ones didn't close get dismissed, so I needed to call it more than once. I think it makes sense to take some time and look at the docs to see if there's some "clear notifications" thing we could run instead